examples/multi-module-sample/build.gradle.kts

```
@@ -1,14 +1,15 @@
+import org.jetbrains.kotlin.gradle.dsl.JvmTarget
 import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
 
 plugins {
   alias(libs.plugins.kotlin)
   id("io.sentry.jvm.gradle")
 }
 
-tasks.withType<KotlinCompile>().configureEach {
-  kotlinOptions {
+tasks.named<KotlinCompile>("compileKotlin") {
```

isn’t the new way to do this something like this?
```suggestion
kotlin {
```